server/src/resolvers/Mutation.js

```diff
@@ -0,0 +1,13 @@
+const mutation = {
```

Comment:
Let's capitalize this property to treat them as a class
```suggestion
const Mutation = {
```